projects/simple-nix/stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-18.27
```

Comment:
Note that all the READMEs also contain text like
> This project uses [lts-18.27](https://www.stackage.org/lts-18.27), which includes Clash 1.4.7. We've added the extra-deps section to make sure Stack fetches the latest version of Clash, 1.6.3, instead.

Could you update those as well?

(It still says 1.6.3 because the other PR is still unmerged)

Reply:
Good point. I've updated the description and removed the `extra-deps`. I'm leaving the bounds + nix stuff for the other PR.